source/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter.cc

```diff
@@ -41,9 +41,50 @@ namespace Envoy {
 namespace Extensions {
 namespace HttpFilters {
 namespace GrpcJsonTranscoder {
-
 namespace {
 
+// TODO(alyssawilk) see if we can get proto to expose this code.
+// If not, move it to a common utility.
+inline std::string CodeEnumToString(ProtobufUtil::error::Code code) {
```

Comment:
return absl::string_view so string is not construct every time?

Reply:
I can, but given I'm hoping to migrate to an existing status code function I'd prefer to not change types.  OK to add a comment and TODO instead?